build_linux.sh

```diff
@@ -4,6 +4,13 @@
 export CC=/usr/bin/gcc-9
 export CXX=/usr/bin/g++-9
 
+cd imageMagick &&
+tar xvf im7.10-linux-source.tar &&
+cd ImageMagick-7.0.10 &&
+./configure --with-quantum-depth=8 --enable-hdri=no --with-x=no --with-utilities=no &&
+sudo make install &&
+cd ../../ &&
```

Comment:
You'll want to extend &&s all the way to the penultimate command. Otherwise travis returns false positives even when it's broken.